VHbbAnalysis/Heppy/test/vhbb.py

```diff
@@ -349,18 +349,36 @@
     "Total"
 ]
 JetAna.factorizedJetCorrections = factorizedJetCorrections
-for jet_corr in factorizedJetCorrections:
-    for sdir in ["Up", "Down"]:
-        name = jet_corr + sdir
-        jetTypeVHbb.variables += [
-            NTupleVariable(
-                "corr_{0}".format(name),
-                lambda x, name = name: getattr(x, 'corr{0}'.format(name), -99),
-                float,
-                mcOnly=True,
-                help=""
-            )
-        ]
+
+for jet_type in [jetTypeVHbb, patSubjetType, subjetcorrType]:
+        for jet_corr in factorizedJetCorrections:
+            for sdir in ["Up", "Down"]:
+                name = jet_corr + sdir
+                jet_type.variables += [
+                    NTupleVariable(
+                        "corr_{0}".format(name),
+                        lambda x, name = name: getattr(x, 'corr{0}'.format(name), -99),
+                        float,
+                        mcOnly=True,
+                        help=""
+                    )
+                ]
+# HTT Subjets
```

Comment:
does it make sense to have subjet stuff in vhbb.py rather than vhbb_combined.py?